### ISSUE TITLE
Installation issue on a blank fresh Windows

### ISSUE THREAD

**Issue by haesleinhuepf** [clEsperanto/pyclesperanto_prototype#36]
Dear all, dear future-self,

after installing pyclesperanto and friends on a fresh new Windows, this line fails:
```
import pyclesperanto_prototype as cle
```

Because:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-1-8cd158322049> in <module>
----> 1 import pyclesperanto_prototype as cle

c:\users\rober\miniconda3\lib\site-packages\pyclesperanto_prototype\__init__.py in <module>
----> 1 from ._tier0 import *
      2 from ._tier1 import *
      3 from ._tier2 import *
      4 from ._tier3 import *
      5 from ._tier4 import *

c:\users\rober\miniconda3\lib\site-packages\pyclesperanto_prototype\_tier0\__init__.py in <module>
      1 from .radius_to_kernel_size import radius_to_kernel_size
      2 from .sigma_to_kernel_size import sigma_to_kernel_size
----> 3 from .create import create
      4 from .create import create_like
      5 from .create import create_pointlist_from_labelmap

c:\users\rober\miniconda3\lib\site-packages\pyclesperanto_prototype\_tier0\create.py in <module>
----> 1 from ._pycl import OCLArray
      2 import numpy as np
      3 
      4 
      5 def create(dimensions):

c:\users\rober\miniconda3\lib\site-packages\pyclesperanto_prototype\_tier0\_pycl.py in <module>
      1 import os
      2 import sys
----> 3 import numpy as np
      4 import pyopencl as cl
      5 from pyopencl import characterize

c:\users\rober\miniconda3\lib\site-packages\numpy\__init__.py in <module>
    303 
    304     if sys.platform == "win32" and sys.maxsize > 2**32:
--> 305         _win_os_check()
    306 
    307     del _win_os_check

c:\users\rober\miniconda3\lib\site-packages\numpy\__init__.py in _win_os_check()
    300                    "See this issue for more information: "
    301                    "https://tinyurl.com/y3dm3h86")
--> 302             raise RuntimeError(msg.format(__file__)) from None
    303 
    304     if sys.platform == "win32" and sys.maxsize > 2**32:

RuntimeError: The current Numpy installation ('c:\\users\\rober\\miniconda3\\lib\\site-packages\\numpy\\__init__.py') fails to pass a sanity check due to a bug in the windows runtime. See this issue for more information: https://tinyurl.com/y3dm3h86
```

**Comment by jni** [comment]
Relevant link:

- https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html

TLDR it's a microsoft compiler problem and should be fixed shortly...

**Comment by haesleinhuepf** [comment]
In case somebody reads that here. This workaround fixed it for me:
```
pip uninstall numpy
pip install numpy==1.19.3
```

**Comment by haesleinhuepf** [comment]
this issue has been fixed with numpy 0.19.5
